quickstart-node/src/services/monday.service.ts

```diff
@@ -0,0 +1,127 @@
+import { ApiClient, ChangeColumnValueOpMutation } from '@mondaydotcomorg/api';
+import {
+  CreateItemMutationVariables,
+  GetItemsInGroupQuery,
+  GetItemsInGroupQueryVariables,
+  GetMeQuery,
+  ItemWithColumnValuesFragment,
+} from 'generated/graphql';
+import { createItem, getItemsInGroup, getMe } from 'queries.graphql';
+import { getValueFromExecutionContext } from '@utils/execution-context.utils';
+import { NotFound } from 'http-errors';
+
+export const UPDATED_MONDAY_API_VERSION = '2024-10';
+
+export class MondayService {
+  private constructor() {}
+
+  static getMondayApiClient(token?: string): ApiClient {
```

Comment:
Please don't use static functions in TS, unless it's a util. Either create objects from classes or simply use exported functions

Reply:
All these functions in this file can be just exported functions. But I want that developers will know explicitly that it is part of `MondayService`. This is also shortens the function names, for example take this function: `isItemColumnValueDifferent`. How can you know we talk about an item from monday board if use it as it is without `MondayService.{functionName}`?